src/Skoruba.IdentityServer4.Admin/Startup.cs

```diff
@@ -36,15 +40,24 @@ public Startup(IHostingEnvironment env)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            services.ConfigureSettingsRoot(Configuration);
+            var settingsRoot = services.BuildServiceProvider().GetService<ISettingsRoot>();
+
             services.AddDbContexts(HostingEnvironment, Configuration);
-            services.AddAuthentication(HostingEnvironment);
+
+            //services.AddAuthentication(HostingEnvironment);
+            new AuthenticationSetup(services, HostingEnvironment, settingsRoot.AppSettings).AddAuthentication();
```

Comment:
Can you please create an extension method for this line? 
Something like `services.AddAuthenticationSetup(HostingEnvironment, settingsRoot.AppSettings);` - It looks much better. :)

Reply:
Extension not fit for DependencyInjection. AuthenticationSetup imade as setup module.